### ISSUE TITLE
Crash when trying to use tilde character in path when trying to import project

### ISSUE THREAD

**Issue by Mai-Lapyst** [eumario/godot-manager#55]
**Describe the bug**
When using the a tilde (~) at the begin of a path the app crashes.
Note: the tilde character is in linux a shorthand for the current users home directory (i.e. `/home/username`).

**To Reproduce**
1. Go to the project manager
2. Click on "Import Project"
3. Put in "Path" a path that starts with the tilde character (~)
4. Click "Add"

**Expected behavior**
That the path is correctly resolved, or a indicator telling the user that the input is wrong.

**Screenshots**
No screenshot but a log:
```
Unhandled Exception:
System.IO.DirectoryNotFoundException: Could not find a part of the path "[godotmanager installdir]/~/projects/github/godot-manager/project.godot".
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.Boolean anonymous, System.IO.FileOptions options) [0x00164] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.IO.FileOptions options) [0x00000] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at (wrapper remoting-invoke-with-check) System.IO.FileStream..ctor(string,System.IO.FileMode,System.IO.FileAccess,System.IO.FileShare,int,System.IO.FileOptions)
  at System.IO.StreamReader..ctor (System.String path, System.Text.Encoding encoding, System.Boolean detectEncodingFromByteOrderMarks, System.Int32 bufferSize) [0x00055] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at System.IO.StreamReader..ctor (System.String path, System.Boolean detectEncodingFromByteOrderMarks) [0x00007] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at System.IO.StreamReader..ctor (System.String path) [0x00000] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at (wrapper remoting-invoke-with-check) System.IO.StreamReader..ctor(string)
  at ProjectConfig.Load (System.String fname) [0x00040] in <f7b093cfd2f94b2ca92bd10d8f1454ab>:0 
  at ProjectFile.ReadFromFile (System.String filePath) [0x0000d] in <f7b093cfd2f94b2ca92bd10d8f1454ab>:0 
  at ImportProject.OnAddBtnPressed () [0x000a5] in <f7b093cfd2f94b2ca92bd10d8f1454ab>:0 

Unhandled Exception:
System.IO.DirectoryNotFoundException: Could not find a part of the path "[godotmanager installdir]/~/projects/github/godot-manager/project.godot".
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.Boolean anonymous, System.IO.FileOptions options) [0x00164] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.IO.FileOptions options) [0x00000] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at (wrapper remoting-invoke-with-check) System.IO.FileStream..ctor(string,System.IO.FileMode,System.IO.FileAccess,System.IO.FileShare,int,System.IO.FileOptions)
  at System.IO.StreamReader..ctor (System.String path, System.Text.Encoding encoding, System.Boolean detectEncodingFromByteOrderMarks, System.Int32 bufferSize) [0x00055] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at System.IO.StreamReader..ctor (System.String path, System.Boolean detectEncodingFromByteOrderMarks) [0x00007] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at System.IO.StreamReader..ctor (System.String path) [0x00000] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at (wrapper remoting-invoke-with-check) System.IO.StreamReader..ctor(string)
  at ProjectConfig.Load (System.String fname) [0x00040] in <f7b093cfd2f94b2ca92bd10d8f1454ab>:0 
  at ProjectFile.ReadFromFile (System.String filePath) [0x0000d] in <f7b093cfd2f94b2ca92bd10d8f1454ab>:0 
  at ImportProject.OnAddBtnPressed () [0x000a5] in <f7b093cfd2f94b2ca92bd10d8f1454ab>:0
```

**Complete the Following:**
 - OS: Linux
 - Version 0.2.1

**Comment by eumario** [comment]
This will also cause an issue on MacOS, which should translate to /Users/<username>, which will also need to be fixed.

**Comment by eumario** [comment]
Fixed with commit [c91860e](https://github.com/eumario/godot-manager/commit/c91860e4c03b65b9d67893c733574d94ed9009eb).
